features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/resources/BitmapCachesManager.kt

```diff
@@ -12,67 +12,120 @@ import android.graphics.Bitmap
 import android.graphics.drawable.Drawable
 import androidx.annotation.MainThread
 import com.datadog.android.api.InternalLogger
+import com.datadog.android.lint.InternalApi
 
-internal class BitmapCachesManager(
-    private val resourcesLRUCache: Cache<Drawable, ByteArray>,
-    private val bitmapPool: BitmapPool,
+/**
+ * Manages the Bitmap caches used by the Session Replay feature.
+ */
+@InternalApi
+class BitmapCachesManager {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏ Why are we doing this ?